Formula/softhsm2.rb

```diff
@@ -0,0 +1,38 @@
+class Softhsm2 < Formula
+  desc "Cryptographic store accessible through a PKCS #11"
+  homepage "https://www.opendnssec.org/softhsm/"
+  url "https://dist.opendnssec.org/source/softhsm-2.3.0.tar.gz"
+  sha256 "5ed604c89a3a6ef9d7d1ee92c28a2c4b3cd1f86f302c808e2d12c8f39aa2c127"
+
+  depends_on "cppunit" => :build
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--with-objectstore-backend-db",
+                          "--with-migrate",
+                          "--with-openssl=" + Formula["openssl"].opt_prefix,
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+
+    system "make", "install"
+
+    include.install "src/lib/pkcs11/cryptoki.h",
+                    "src/lib/pkcs11/pkcs11.h",
+                    "src/lib/pkcs11/pkcs11f.h",
+                    "src/lib/pkcs11/pkcs11t.h"
+  end
+
+  def post_install
+    (var/"lib/softhsm/softhsm").mkpath
+  end
+
+  test do
+    system "#{bin}/softhsm2-util", "--init-token", "--free",
```

Comment:
The test needs to do something other than check for a 0 exit code here.

Reply:
what do you recommend? I have no idea what else I could do.

Reply:
I'm getting
```
bash-3.2$ /usr/local/Cellar/softhsm2/2.3.0/bin/softhsm2-util --init-token --free --label test --so-pin 1234 --pin 1234
Slot 0 has a free/uninitialized token.
ERROR rv=0x00000030: Could not initialize the token.
Please check log files for additional information.
bash-3.2$ echo $?
1
bash-3.2$
```
So not sure it's working as expected right now.